extraction/src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipelineBuilder.scala

```diff
@@ -62,17 +62,17 @@ class ExtractionPipelineBuilder(batchSize: Long, getClient: () => EncodeClient)
       .flatMap(_.getValue.fold(throw _, _.read[Array[Msg]]("@graph")))
 
   override def buildPipeline(ctx: ScioContext, args: Args): Unit = {
-    /**
```

Comment:
Not scalafmt: Scaladoc-style comments (double ** at the front) are only valid on top-level members in a class. If they're used on nested methods like this one, they're marked as "unmoored" and cause packaging to fail.